jupyterlab-dataops/Dockerfile

```diff
@@ -40,6 +36,21 @@ COPY resources/jupyter/ /etc/jupyter/
 # Expose port for Vertex AI compatibility
 EXPOSE 8080
 
-FROM base as bigquery
+# use default cloudbeaver in non-bigquery target
+FROM base AS default-stage
```

Comment:
I would suggest to add some prefix that this stage change dbeaverconfig 
- `default-dbeaver`
- `bigquery-dbeaver`